app/views/pages/induction_tutor_materials/ambition/year_two.html.erb

```diff
@@ -0,0 +1,110 @@
+<% content_for :before_content, govuk_back_link(text: 'Back', href: current_user ? dashboard_path : root_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-9">Ambition Institute's handbooks and training outlines</h1>
+
+    <h2 class="govuk-heading-l govuk-!-margin-bottom-3">Handbooks</h2>
+
+    <ul class="govuk-list govuk-list--spaced">
+      <li>
+        <%= govuk_link_to "ECF Programme Handbook (PDF, 107 KB, 10 pages)",
+                          href = "https://www.early-career-framework.education.gov.uk/ambition/wp-content/uploads/sites/3/2020/08/Ambition-EarlyCareerTeachers_2020_ProgrammeHandbook.pdf" %>
+      </li>
+      <li>
+        <%= govuk_link_to "ECF Lead Handbook (PDF, 102 KB, 12 pages)",
+                          href = "https://www.early-career-framework.education.gov.uk/ambition/wp-content/uploads/sites/3/2020/09/Ambition-EarlyCareerTeachers_2020_LeadHandbook_Guidebook_Edit-1-1.pdf" %>
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-l govuk-!-margin-top-6 govuk-!-margin-bottom-3">Training outlines</h2>
+
+    <p class="govuk-!-margin-bottom-3">Here is a placholder of contextual blurb about training outlines. Lorem ipsum
+      dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+      aliqua. </p>
+
+    <%= render SubnavComponent.new do |component| %>
+      <%= component.nav_item(path: induction_tutor_materials_path(provider: "ambition", year: "year-one")) do %>
+        Year 1
+      <% end %>
+      <%= component.nav_item(path: induction_tutor_materials_path(provider: "ambition", year: "year-two")) do %>
+        Year 2
+      <% end %>
+    <% end %>
```

Comment:
If it ends up being used on several pages, probably worth looking into commonising it. Either with a partial, or a component :)